packages/pieces/community/rabbitmq/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { createPiece, PieceAuth, Property } from '@activepieces/pieces-framework';
+import { messageReceived } from './lib/triggers/message-received';
+import { sendMessageToExchange } from './lib/actions/send-message-to-exchange';
+import { sendMessageToQueue } from './lib/actions/send-message-to-queue';
+
+export const rabbitmqAuth = PieceAuth.CustomAuth({
+  description: "Rabbitmq Auth",
+  required: true,
+  props: {
+    host: Property.ShortText({
+      displayName: "Host",
+      description: "Host",
+      required: true,
+    }),
+    username: Property.ShortText({
+      displayName: "Username",
+      description: "Username",
+      required: true,
+    }),
+    password: PieceAuth.SecretText({
+      displayName: "Password",
+      description: "Password",
+      required: true,
+    }),
+    port: Property.Number({
+      displayName: "Port",
+      description: "Port",
+      required: true,
+    }),
+  },
+});
+
+export const rabbitmq = createPiece({
+  displayName: "Rabbitmq",
```

Comment:
nit: can you please change to RabbitMQ

That's how they write it on their website